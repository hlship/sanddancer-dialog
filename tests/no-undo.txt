open glove
get pack
smoke pack
undo
i
open wallet
undo
i

