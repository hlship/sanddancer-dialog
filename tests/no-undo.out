

but it's too late, you're crashing, you crash...

Middle of Nowhere (in the pickup truck)
You've smashed up against a tall Saguaro, which is all you can see out the
windshield except a few feet of desert sand, then blackness. Hell, you must have
driven off the road and crashed. Ocean keeps telling you this night commute on
this lonely highway's gonna kill you; maybe you should start listening.

On the dash you can see a piece of jade.

The glove compartment is closed.

Through the windshield, you see a little lizard clinging frantically to the
Saguaro.

Something about the piece of jade makes you want to brood about your grandma's
stories.

> open glove
You open the glove compartment, revealing your pack of cigarettes.

> get pack
You take your pack of cigarettes out of the glove compartment.

> smoke pack
You pull out a cigarette and flick it into life. Thoughts tumble through your
head.

Well, you're stuck here. It's dark outside the beam of your headlights. Well,
you could start by getting out of this truck.

And you still haven't quit smoking. You toss the the butt out the window of the
truck and wonder what to do next.

> undo
Yeah, you wish you could unsmoke all the cigs you've inhaled. Life doesn't work
like that.

> i
You have your pack of cigarettes, your lighter, your wallet, and your emotional
baggage. You're wearing your denim jacket.

In your emotional baggage are your grandma's stories.

> open wallet
You open your wallet, revealing your driver's license, a receipt from Big
Jimmy's, and a folded photo.

Something about the receipt from Big Jimmy's makes you want to brood about your
shit job.

> undo
Undoing the last turn (open wallet).
Middle of Nowhere (in the pickup truck)

> i
You have your pack of cigarettes, your lighter, your wallet, and your emotional
baggage. You're wearing your denim jacket.

In your emotional baggage are your grandma's stories.

> 
